src/main/resources/languages/messages_ar.yml

```diff
@@ -255,7 +255,7 @@ messages:
     whirlwind: "&a&oأبعدت تعويذتك القذيفة عنك"
     wizard: "&a&oحسنت تعويذتك حالة حظك لكن غالبا سوءت حالات أخرى"
   unknown-player: "&4لاعب غير معروف: &c%player%"
-  unlocked: &bتم فتح &7"%research%"
+  unlocked: '&bتم فتح&7"%research%"'
```

Comment:
Looks like there's a fucked up speech mark though they all use it so I'd replace the speech mark with quoation marks to keep this consistent.